docs/folder-structure.md

```diff
@@ -0,0 +1,96 @@
+# F36 Folder Structure
+
+To make it easy to maintain our files and folders we ask every contributor to follow this folder structure.
+Every package has its own folder, except for components which should be kept in the folder `/components` (every component must have its own folder).
+
+## Table of contents
+
+- [Components](#components)
+  - [Packages with more than one component](#packages-with-more-than-one-component)
+  - [Packages with utility functions](#packages-with-utility-functions)
+  - [Why do we use MDX?](#why-do-we-use-mdx)
+  - [Why is every `.mdx` file named "README"](#why-is-every-mdx-file-named-readme)
+
+## Components
+
+The basic folder structure of a component looks like this:
+
+```
+/my-component
+  /src
+    index.ts
+    MyComponent.tsx
+    MyComponent.styles.ts
+  /stories
+    # stories for storybook
+  /test
+    # test files
+  README.mdx
+  package.json
+```
+
+### Packages with more than one component
+
+Sometimes the package will include a "base" component and some variants that are build from the base component (e.g. Button).
+In these cases, the folder structure of the package will look more like this:
+
+```
+/button
+  /src
+    /ButtonGroup
+      README.mdx
+      index.ts
+      ButtonGroup.tsx
+    /IconButton
+      README.mdx
+      index.ts
+      IconButton.tsx
+    /ToggleButton
+      README.mdx
+      index.ts
+      IconButton.tsx
+    index.ts
+    Button.tsx
+  /stories
+    # stories for storybook
+  /test
```

Comment:
I am wondering if maybe the tests folder should live in each component folder? but maybe it's overkill to have folder `test` in every component?

Reply:
I like when we have the test file close to the component file, like:

```
Button.tsx
Button.test.ts
```

But since we were creating a `test` folder, I put in the document the way we do it now
Let's see what everyone thinks

Reply:
We can have the `.test.tsx` files in each component folder if needed.

Reply:
Yeah, ok. I guess we need to keep it as it is (do we? 😁). But I'm with you, I feel like tests belong to each component.

Reply:
I will open a poll in the slack channel
Since, you, @massao,  and I want to change this already hahahaha

Reply:
I will merge this now and we change this part about the tests once everyone has the chance to express their opinion